src/test/denialofservice_tests.cpp

```diff
@@ -223,47 +223,84 @@ BOOST_AUTO_TEST_CASE(peer_discouragement)
 {
     const CChainParams& chainparams = Params();
     auto banman = MakeUnique<BanMan>(GetDataDir() / "banlist.dat", nullptr, DEFAULT_MISBEHAVING_BANTIME);
-    auto connman = MakeUnique<CConnman>(0x1337, 0x1337);
+    auto connman = MakeUnique<CConnmanTest>(0x1337, 0x1337);
```

Comment:
std::make_unique for new code?

Also, could split the refactoring changes from the new-test changes?

Reply:
Nvm, forget the std::make_unique suggestion. We want to backport the test changes.